dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/DataRelationsValidator.java

```diff
@@ -322,12 +321,13 @@ private boolean isAOCForCOs(CategoryOptionCombo aoc, Set<CategoryOption> categor
   }
 
   private Program getEnrollmentProgramFromEvent(TrackerBundle bundle, Event event) {
-    Enrollment preheatEnrollment = bundle.getPreheat().getEnrollment(event.getEnrollment());
+    Enrollment preheatEnrollment =
+        bundle.getPreheat().getEnrollment(event.getEnrollment().getValue());
```

Comment:
Do we do this often? If so add support for `getYYY(UID`

Reply:
I am going to change this when I tackle enrollments.